apps/nextjs-app/src/features/app/blocks/view/kanban/components/KanbanContainer.tsx

```diff
@@ -190,7 +191,7 @@ export const KanbanContainer = () => {
             );
           }}
         </Droppable>
-        {stackCreatable && isSingleSelectField && (
+        {stackCreatable && isSingleSelectField && !isEmptyStackHidden && (
```

Comment:
The create button should not be hidden when hide empty stack.

Reply:
@Sky-FE Won't it be confusing for users, when they try to create new kanban, and it doesn't appear (because it's empty)?

Reply:
@kpodp0ra 
In fact, this is just a way to quickly add single-select options. I will optimize the experience in the future, but for now, the create button should not be hidden, because this is only at the interactive level and does not involve permissions.

Reply:
@Sky-FE Sure